membership-attribute-service/conf/DEV.public.conf

```diff
@@ -11,28 +11,18 @@ touchpoint.backend.test=UAT
 
 stage=DEV
 
+play.filters.cors.supportsCredentials = true
```

Comment:
This might be defaulted to true, but to save confusion, I thought it should be hardcoded anyway.